routes/clucks.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express");
+const knex = require("../db/client");
+const router = express.Router();
+
+ //Render index template
+ router.get("", (req, res) => {
+  res.redirect("/clucks")
+  });
+
+  // render clucks index
+  router.get("/clucks", (req, res) => {
+    knex("clucks")
+      .orderBy("created_at", "desc")
+      .then(clucks => {
+        res.render("clucks/index", { clucks });
+      });
+  });
+
+
+    //Render sign-in template
+  router.get("/clucks/sign_in", (req, res) => {
```

Comment:
you already have `sign_in` from root. what's the need to duplicate the endpoint?